crates/bevy_pbr/src/wireframe.rs

```diff
@@ -52,21 +52,24 @@ impl Plugin for WireframePlugin {
     }
 }
 
-fn extract_wireframes(mut commands: Commands, query: Extract<Query<Entity, With<Wireframe>>>) {
-    for entity in &query {
-        commands.get_or_spawn(entity).insert(Wireframe);
+fn extract_wireframes(mut commands: Commands, query: Extract<Query<(Entity, &Wireframe)>>) {
+    for (entity, wireframe) in &query {
+        commands.get_or_spawn(entity).insert(wireframe.clone());
```

Comment:
If it's forcibly disabled in the `Wireframe` component, this shouldn't be extracted when it's false.

Reply:
Actually, it should be extracted both when true or false. If it isn't, then in the case where `global: true` and `Wireframe {enable: false}`, it would still draw the wireframe, even though we said we didn't want it for this specific entity.

To verify, I tried it with: 
```rust
fn extract_wireframes(mut commands: Commands, query: Extract<Query<(Entity, &Wireframe)>>) {
    query
        .iter()
        .filter(|(_, wireframe)| wireframe.enable)
        .for_each(|(entity, wireframe)| {
            commands.get_or_spawn(entity).insert(wireframe.clone());
        });
}
```
and indeed, in the example it would draw the plane with a wireframe, even though it is given a `Wireframe: false`.